Packs/XSOARmirroring/Integrations/XSOARmirroring/XSOARmirroring_test.py

```diff
@@ -131,6 +137,52 @@ def test_fetch_incidents(mocker):
     assert dateparser.parse(next_run['last_fetch']) == dateparser.parse(INCIDENTS[-1]['created']) + timedelta(milliseconds=1)
 
 
+def test_fetch_incidents_mirroring_playbook_id(mocker):
+    """
+    Given:
+        - List of incident.
+
+    When:
+        - Running the fetch_incidents and getting this incident.
+
+    Then:
+        - Ensure the incident result contains playbookId field as expected.
+    """
+    mocker.patch.object(Client, 'search_incidents', return_value=INCIDENTS_MIRRORING_PLAYBOOK_ID)
+
+    first_fetch = dateparser.parse('3 days').strftime(XSOAR_DATE_FORMAT)
+    client = Client("")
```

Comment:
```suggestion
    client = Client("dummy token")
```
so future maintainers know what this empty string means